includes/sanitizers/class-amp-tag-and-attribute-sanitizer.php

```diff
@@ -127,17 +145,41 @@ public function get_scripts() {
 		return $scripts;
 	}
 
+	/**
+	 * Process alternative names in attribute spec list.
+	 *
+	 * @since 0.7
+	 *
+	 * @param array $attr_spec_list Attribute spec list.
+	 * @return array Modified attribute spec list.
+	 */
+	private function process_alternate_names( $attr_spec_list ) {
+		foreach ( $attr_spec_list as $attr_name => &$attr_spec ) {
+			if ( '[' === $attr_name[0] ) {
+				$placeholder_attr_name = $this->args['amp_bind_placeholder_prefix'] . trim( $attr_name, '[]' );
+				if ( ! isset( $attr_spec[ AMP_Rule_Spec::ALTERNATIVE_NAMES ] ) ) {
+					$attr_spec[ AMP_Rule_Spec::ALTERNATIVE_NAMES ] = array();
+				}
+				$attr_spec[ AMP_Rule_Spec::ALTERNATIVE_NAMES ][] = $placeholder_attr_name;
+			}
+
+			// Save all alternative names in lookup to improve performance.
+			if ( isset( $attr_spec[ AMP_Rule_Spec::ALTERNATIVE_NAMES ] ) ) {
```

Comment:
I thought about that, but there are _other_ alternative names than just the bind ones. For example, `srcset` is an alt for `src`, and `xlink:href` is an alternate for `href`. These need to be accounted for as well. So that is why there is a separate conditional.